salt/metalk8s/volumes/prepared/files/metalk8s-volume-clean-up

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import os
+import subprocess
+import sys
+
+
+def die(message):
+    print(message, file=sys.stderr)
+    sys.exit(1)
+
+
+def get_volume_id():
+    if len(sys.argv) != 2:
+        die("Invalid number of arguments: expected only one volume ID")
+
+    return sys.argv[1]
+
+
+def find_device(volume_id):
+    path_by_uuid = "/dev/disk/by-uuid/{}".format(volume_id)
+    if os.path.exists(path_by_uuid):
+        return path_by_uuid
+
+    path_by_partuuid = "/dev/disk/by-partuuid/{}".format(volume_id)
+    if not os.path.exists(path_by_partuuid):
+        die("Volume '{}' backing device was not found".format(volume_id))
+
+    partition_name = os.path.basename(os.path.realpath(path_by_partuuid))
+    device_name = os.path.basename(
+        os.path.realpath("/sys/class/block/{}/..".format(partition_name))
+    )
+
+    return "/dev/{}".format(device_name)
```

Comment:
Add an assertion to check this actually exists? It always should, but shouldn't harm...

Reply:
Done (and also checking that this is a loop device)